website/content/docs/overview/architecture/index.md

```diff
@@ -9,25 +9,57 @@ weight: 10
 The information below is outdated
 {{</ warning >}}
 
-{{< name >}} takes a **modular** approach to provide an open platform for controlling and securing the use of data across an organization. The figure below showcases the current architecture of the {{< name >}} platform, running on top of OpenShift Container Platform. The storage systems shown in the lower half of the figure are merely an example.
+{{< name >}} takes a **modular** approach to provide an open platform for controlling
+ and securing the use of data across an organization. The figure below showcases the
+  current architecture of the {{< name >}} platform, running on top of Kubernetes. 
+ The storage systems shown in the lower half of the figure are merely an example.
 
+The core parts of {{< name >}} are based on [Kubernetes operators](https://www.openshift.com/learn/topics/operators) 
+ and custom resource definition in order to define its work items and reconcile the state of
+ them.
 
-![Architecture](architecture.png)
+The primary interaction object for a data user is the `M4DApplication` CRD where a user defines which
+data should be used for which purpose. The following chart and description describe the architecture and components of
+{{< name >}} relative to when they are used.
 
-{{< bullet n=1 >}} A {{< name >}} operator configures the resources available to {{< name >}}: compute and storage resources used to run governance actions and optimize data access performace, and `M4DModule` resources that describe modules injectable into the data path.
+![Architecture](workflow_multicluster.png)
 
-{{< bullet n=2 >}} A data streward configures policies in an external policy manager over assets defined in an external data catalog. The policy manager and data catalog are connected to {{< name >}} `PolicyCompiler` and `DataCatalog` services via `PolicyManagerAdapter` and `DataCatalogAdapter`, respectively.
+Before the data user can perform any actions a data operator has to [install](../../setup/quickstart) the {{< name >}} and modules
+that can be used by the user. [Modules](../modules) are a way that {{< name >}} abstracts data access for
```

Comment:
I would remove the words "that can be used by the user."  The data user doesn't really use modules directly.  They are invoked indirectly by the mesh.

Also, instead of "modules are a way" I would suggest "modules are an extensible mechanism for processing data."

Reply:
The user does not initiate modules. But e.g. the read and write modules are the point of access for the user. (e.g. arrow flight module)